twilio/modules/listener/tests/test.bal

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/lang.'int as ints;
+import ballerina/lang.runtime;
+import ballerina/log;
+import ballerina/os;
+import ballerina/test;
+import ballerina/http;
+
+configurable string & readonly twilioAccountSid = os:getEnv("ACCOUNT_SID");
+configurable string & readonly twilioAuthToken = os:getEnv("AUTH_TOKEN");
+configurable string & readonly fromNumber = os:getEnv("SAMPLE_FROM_MOBILE");
+configurable string & readonly toNumber = os:getEnv("SAMPLE_TO_MOBILE");
+configurable string & readonly test_message = os:getEnv("SAMPLE_MESSAGE");
+configurable string & readonly twimlUrl = os:getEnv("TWIML_URL");
+configurable string & readonly callbackUrl = os:getEnv("CALLBACK_URL");
+configurable string & readonly port = os:getEnv("PORT");
+
+int PORT = check ints:fromString(port);
+listener Listener twilioListener = new (PORT, twilioAuthToken, callbackUrl);
+
+@test:BeforeSuite
+isolated function beforeSuit() {
+    log:printInfo("Listener Started");
+    runtime:sleep(10);
+}
+
+service / on twilioListener {
+    isolated remote function onSmsSent(SmsStatusChangeEvent event) returns error? {
+        log:printInfo("Received onSmsSent-message ", eventMsg = event);
+    }
+    isolated remote function onSmsDelivered(SmsStatusChangeEvent event) returns error? {
+        log:printInfo("Received onSmsDelivered-message ", eventMsg = event);
+    }
+    isolated remote function onCallRang(SmsStatusChangeEvent event) returns error? {
+        log:printInfo("Received onCallRang-message ", eventMsg = event);
+    }
+    isolated remote function onCallCompleted(SmsStatusChangeEvent event) returns error? {
+        log:printInfo("Received onCallCompleted-message ", eventMsg = event);
+    }
+}
+
+http:Client httpClient = checkpanic new("http://localhost:9000/onChange");
```

Comment:
What is the reason for using `checkpanic`?

Reply:
We we can avoid checkpanic. This is addressed addressed in the PR#96. These test files are no longer available in the master branch.